File: 168-excel-sheet-column-title/168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, n: int) -> str:
        res=""
        while n>0:
            d = n%26
            if d==0:
                res = 'Z'+res
                n = n//26 - 1
            else:
                res = chr(64+d)+res
                n=n//26
        return res

File: 168-excel-sheet-column-title/168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String res="";
        while(columnNumber>0)
        {
            int d = columnNumber%26;
            if(d==0){
                res = 'Z'+res;
                columnNumber=columnNumber/26-1;
            }
            else
            {   
                res = (char)(64+d)+res;
                columnNumber/=26;
            }
        }
        return res;
    }
}